tekton-pipeline/task-bump-jenkins.yaml

```diff
@@ -0,0 +1,105 @@
+﻿apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  creationTimestamp: '2020-10-15T08:58:13Z'
+  generation: 17
+  managedFields:
+    - apiVersion: tekton.dev/v1beta1
+      fieldsType: FieldsV1
+      fieldsV1:
+        'f:spec':
+          .: {}
+          'f:params': {}
+          'f:results': {}
+          'f:steps': {}
+          'f:volumes': {}
+      manager: Mozilla
+      operation: Update
+      time: '2020-10-20T09:19:22Z'
+  name: bump-jenkins
+  namespace: art-jenkins
+  resourceVersion: '66517529'
+  selfLink: /apis/tekton.dev/v1beta1/namespaces/art-jenkins/tasks/bump-jenkins
+  uid: 1ba0aa05-35ff-4ea3-8835-612dd692aa33
+spec:
+  params:
+    - name: jenkins-version
+      type: string
+    - name: ocp-branch
+      type: string
+  results:
+    - description: the result of this task
+      name: jobresult
+  steps:
+    - image: docker.io/library/centos
```

Comment:
I would like this to be `art-ci-toolkit`, which in addition to doozer/elliott should include scripts necessary for this task.

Similar for the other tasks.

Reply:
It would be nice to have a base image to construct with.